tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+
+1. Update package lists to get information on the newest packages and their dependencies:
+
+	```bash
+	sudo apt-get update
+	```
+
+2. Install Apache server and its module for PHP support:
+
+	```bash
+	sudo apt-get install apache2 apache2-mod-php7.0 -y
+	```
+
+	After the installation is completed, open your browser and visit http://`[SERVER_EXTERNAL_IP]` (note that it is a HTTP link instead of a HTTPS link). `[SERVER_EXTERNAL_IP]` can be found on the [VM Instances](https://console.cloud.google.com/compute/instances) page, in the row of the instance that you connect to. You should see a page titled “Apache Debian Default Page” in your browser. 
+
+3. Install MariaDB server:
+
+	```bash
+	sudo apt-get install mariadb-server -y
+	```
+
+4. Set up MariaDB:
+
+	```bash
+	sudo mysql_secure_installation
+	```
+
+	The script prepares MariaDB for production use. For security reasons you should follow the prompts on screen, set up a root password, remove all anonymous users, disable remote login for root user, remove test database and reload the privilege tables.
+
+5. Create a database and a database user for XenForo to use:
+
+	```bash
+	sudo mysql
+	```
+
+	Then enter the following commands in MariaDB monitor:
+
+	```sql
+	CREATE DATABASE [DATABASE_NAME];
+	CREATE USER ‘[DATABASE_USER]’@’localhost’ IDENTIFIED BY ‘[DATABASE_PASSWORD]’;
+	GRANT ALL PRIVILEGES ON *.* TO ‘[DATABASE_USER]’@’localhost’;
+	```
+
+	Commands above create a new database and a new database user with full privileges in the system. Replace `[DATABASE_NAME]`, `[DATABASE_USER]` and `[DATABASE_PASSWORD]` with values of your own. You may want to write down those values as you need to use them later when installing XenForo.
+
+6. Install PHP and PHP modules required by XenForo:
+
+	```bash
+	sudo apt-get install php7.0 php7.0-mysql php7.0-gd php7.0-xml -y
+	```
+
+7. Install additional tools and packages:
+
+	```bash
+	sudo apt-get install unzip postfix libsasl2-modules
+	```
+
+	Not all of the tools and packages listed above are required XenForo; however you may find them useful when installing and configuring XenForo.
+
+	During the installation of postfix a configuration screen will come up. Use TAB key to move the cursor, UP and DOWN keys to change the choice and Enter to confirm. For this tutorial we will use the `Local Only` type. You can leave other settings unchanged.
+
+	Now the instance is ready for running XenForo. 
+
+## Uploading and Configuring XenForo
+
+1. Copy the XenForo installation file to your Compute Engine instance:
+
+	* Open [Cloud Storage Browser](https://console.cloud.google.com/storage/browser) page and click Create **Bucket**. Enter a name for the new bucket and choose a storage class for the bucket in the new page. For this tutorial you should create a regional bucket in the same location as your Compute Engine instance.
```

Comment:
"Visit the [Cloud Storage Browser](https://console.cloud.google.com/storage/browser) menu in the Cloud Platform Console"